lesson24/index.js

```diff
@@ -0,0 +1,46 @@
+const checkboxLink = document.getElementById("js-checkbox-link");
+const modalCloseButton = document.getElementById("js-modal-close-button");
+const modalInner = document.getElementById("js-modal-inner");
+const submitButton = document.getElementById("js-submit-button");
+
+const openModal = () => {
+    document.getElementById("js-modal-area").classList.add("is-active");
+};
+
+const closeModal = () => {
+    document.getElementById("js-modal-area").classList.remove("is-active");
+};
```

Comment:
nits
For a single line, {} can be omitted.
However, as other processes may be added in the future, the modification is left to you.
```suggestion
const openModal = () => document.getElementById("js-modal-area").classList.add("is-active");

const closeModal = () => document.getElementById("js-modal-area").classList.remove("is-active");
```

Reply:
Thank you for your review!
I did it again, even though you pointed it out before 😨.
Fixed it so it's less lines and simpler to read! Thank you!
[567ccc6](https://github.com/haru-programming/JavaScript-lessons/pull/28/commits/567ccc6a43b8eb26b04a28362a96ce85f266f2d5)